src/TodoListItem.js

```diff
@@ -0,0 +1,21 @@
+//rfc
+//props - from parent to child
+
+/** JSDoc
+ * 
+ * @param {*} props 
+ * @returns 
+ */
+
+import React from 'react';
+
+export default function TodoListItem(props) {
+
+    
+    return (
+        <div>
+            <li>{props.title}</li>
+        </div>
```

Comment:
Better don't use `div` as a parent element for returning `li`. You should either to use `Fragment` or remove `div` tag and just leave `li` as a parent tag because you are returning one line. Look at the documentation of React https://ru.react.js.org/docs/fragments.html.